dbt/adapters/glue/impl.py

```diff
@@ -104,22 +104,24 @@ def get_connection(self):
 
     def list_schemas(self, database: str) -> List[str]:
         session, client = self.get_connection()
-        responseGetDatabases = client.get_databases()
-        databaseList = responseGetDatabases['DatabaseList']
+        paginator = client.get_paginator('get_databases')
         schemas = []
-        for databaseDict in databaseList:
-            databaseName = databaseDict['Name']
-            schemas.append(databaseName)
+        for page in paginator.paginate():
+            databaseList = page['DatabaseList']
+            for databaseDict in databaseList:
+                databaseName = databaseDict['Name']
+                schemas.append(databaseName)
         return schemas
 
     def list_relations_without_caching(self, schema_relation: SparkRelation):
         session, client = self.get_connection()
         relations = []
+        tables = []
+        paginator = client.get_paginator('get_tables')
         try:
-            response = client.get_tables(
-                DatabaseName=schema_relation.schema,
-            )
-            for table in response.get("TableList", []):
+            for table in paginator.paginate(DatabaseName=schema_relation.schema):
```

Comment:
nit: this is not really **table**, but a list of tables. I suggest to change the name to something more meanifull e.g. tables_page.

Reply:
Done. Valid nit.